content/blog/2014/sqlalchemy-postgres-autocommit.md

```diff
@@ -20,6 +20,21 @@ summary: |
 date: 2014-06-14
 ---
 
+{% import 'utility.macros.njk' as utility %}
+
+{% set update = ['Update', utility.datetime('2024-04-26')] | join(' ') %}
+{% callout 'note', update %}
+This article was written before the release of SQLAlchemy 2.0. The library has
+changed significantly since then, and we have added relevant notes to the
+[autocommit mode] and [real autocommit] sections. TL;DR -- autocommit mode was
+deprecated and the isolation level approach is now recommended and thoroughly
+explained by the official docs.
+
+[autocommit mode]: #sqlalchemy-s-autocommit-mode-not-what-you-think
+[real autocommit]: #turning-on-real-autocommit
```

Comment:
Do we want to add the "Starting a Transaction" section here too?

Reply:
Fixed in a71cf400abf0d29b2fc7a9d9e9da8125bfc0d8de